frontend/ui/widget/dictquicklookup.lua

```diff
@@ -331,6 +420,19 @@ function DictQuickLookup:changeDictionary(index)
     self.dictionary = self.results[index].dict
     self.lookupword = self.results[index].word
     self.definition = self.results[index].definition
+    self.fullpage = self.results[index].fullpage
+    -- add "dict_index / nbresults" to displayword if not fullpage,
+    -- so we know where we're at and what's yet to see
+    if self.fullpage then
+        self.displayword = self.lookupword
+    else
+        self.displayword = self.lookupword.."   "..index.." / "..#self.results
+    end
+    -- add queried word to 1st result's definition, so we can see
+    -- what was the selected text and if we selected wrong
+    if index == 1 and not self.fullpage then
```

Comment:
The `not self.fullpage` branch here can be folded into previous else branch so you are not checking it twice.